tools/tsconfig.lib.json

```diff
@@ -6,6 +6,6 @@
     "outDir": "../dist/out-tsc",
     "types": ["node"]
   },
-  "exclude": ["**/*.spec.ts"],
+  "exclude": ["**/*.spec.ts", "**/*.test.ts"],
```

Comment:
based on feedback nx now sets also `test.ts` for testing globes - this was done by auto migration